examples/erc20/DeFi/DeFiMoneyMarket/mUSDC.xml

```diff
@@ -61,7 +61,8 @@
                     <ts:string xml:lang="en">Amount to supply</ts:string>
                 </ts:name>
                 <ts:origins>
-                    <ts:user-entry as="uint"/>
+                    <!--USDC has 6 decimals -->
```

Comment:
@coreycaplan3 could you please use the JavaScript that I used to make this happen and then use e18? We don't have e6